.github/workflows/pull_request_windows_build.yml

```diff
@@ -9,6 +9,7 @@ on:
       - stage
       - ballerina-[0-9]+.[0-9]+.x
       - stage-slbeta4
+      - langlib-fixes
```

Comment:
Let's remove this once https://github.com/ballerina-platform/ballerina-lang/pull/33542 is merged